yarpl/include/yarpl/Refcounted.h

```diff
@@ -270,17 +306,26 @@ bool operator>=(std::nullptr_t, const Reference<T>& rhs) noexcept {
 
 template <typename T, typename... Args>
 Reference<T> make_ref(Args&&... args) {
-  return Reference<T>(new T(std::forward<Args>(args)...));
+  return Reference<T>(
+    new T(std::forward<Args>(args)...),
+    detail::skip_initial_refcount_check{}
+  );
 }
 
 template <typename T>
 Reference<T> get_ref(T& object) {
-  return Reference<T>(&object);
+  return Reference<T>(
```

Comment:
Merge the `enable_if` part in the other diff to here. 
We should not be calling `get_ref` on a type T, if T is not a Refcounted object.